dashboard/config/scripts/levels/CSD U3L04 TFMD_pilot.level

```diff
@@ -11,7 +11,7 @@
     "parent_level_id": 15984,
     "name_suffix": "_pilot",
     "encrypted": "false",
-    "teacher_markdown": "[Shapes and Parameters](https://docs.code.org/concepts/game-lab/shapes-and-parameters/) - (Map Level - [download as pdf](https://drive.google.com/a/code.org/uc?id=1qjYMu7Kvs-sKyAVGM2c_6NK3S6_Lo7Pm&export=download))"
+    "teacher_markdown": "[Shapes and Parameters](https://studio.code.org/docs/concepts/game-lab/shapes-and-parameters/) - (Map Level - [download as pdf](https://drive.google.com/a/code.org/uc?id=1qjYMu7Kvs-sKyAVGM2c_6NK3S6_Lo7Pm&export=download))"
```

Comment:
changing these links which point to concepts are actually introducing a temporary regression, since this content will go from having a working sidebar to not having one.  but since there are only two of these, and it's part of our plan to fix many other just like these, I think this is ok.